css/contact.css

```diff
@@ -0,0 +1,251 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+body {
+  margin: 0;
+  padding: 0;
+}
+h1,
+h2,
+p {
+  font-family: "Roboto";
+}
+h1 {
+  font-size: 100px;
+}
+/* HEADER */
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: white;
+  color: black;
+  display: flex;
+  align-items: center;
+  height: 120px;
+  z-index: 3;
+}
+
+.header .left {
+  flex: 1;
+  padding-left: 20px;
+}
+.left {
+  display: flex;
+  align-items: center;
+}
+.left p {
+  margin-right: 10px;
+}
+#box {
```

Comment:
Great use of a mixture of classes and IDs to target different elements for styling! Nice job! 🥇